pkg/controller/machine/controller.go

```diff
@@ -89,9 +91,13 @@ func AddWithActuator(mgr manager.Manager, actuator Actuator) error {
 	return AddWithActuatorOpts(mgr, actuator, controller.Options{})
 }
 
-func AddWithActuatorOpts(mgr manager.Manager, actuator Actuator, opts controller.Options) error {
+func AddWithActuatorAndFeatureGates(mgr manager.Manager, actuator Actuator, gate featuregate.MutableFeatureGate) error {
+	return AddWithActuatorOpts(mgr, actuator, controller.Options{}, gate)
+}
+
+func AddWithActuatorOpts(mgr manager.Manager, actuator Actuator, opts controller.Options, gate ...featuregate.MutableFeatureGate) error {
```

Comment:
I think this should probably just be removed. We don't want some instances where we can add an operand with feature gates, and then one without.

Reply:
It also makes the contract less fragile (`gate[0]` below)